master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Migrate from one IP pool to another
+---
+
+### Big picture
+
+Migrate pods from one IP pool to another on a running cluster without network disruption.
+
+### Value
+
+Pods are assigned IP addresses from IP pools that you configure in {{site.prodname}}. As pods increase, you may want pods to move a larger CIDR. Or, you may need to move pods from a CIDR that was used by mistake. {{site.prodname}} lets you migrate from one IP pool to another one on a running cluster without network disruption. 
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **IPPool** resource 
+
+### Concepts
+
+#### IP pools and cluster CIDRs
+
+{{site.prodname}} supports changing IP pools for pods if the IP pool you are changing to is within the same cluster CIDR. Although it is technically feasible to change a pod to an IP pool outside the cluster CIDR, we do not recommend it; this requires restarting components (affecting application availability), and if you change to a CIDR range that doesn’t include the already-assigned value, existing pods will lose connectivity.
+
+### Before you begin...
+
+- You must be using {{site.prodname}} IPAM. 
+
+  To verify, ssh to one of your Kubernetes nodes and view the CNI configuration.  
+
+  `cat /etc/cni/net.d/10-calico.conflist`
+
+  Look for the "type" entry:
+
+  <pre>
+     "ipam": {
+           "type": "calico-ipam"
+      }, 
+  </pre>
+
+   If the type is “calico-ipam,” you are good to go. If the IPAM is set to something else, or the 10-calico.conflist file does not exist, you cannot use this feature in your cluster. 
```

Comment:
I think the comma should be outside of the quote here, right? `"calico-ipam",` rather than `"calico-ipam,"`

Reply:
US/American is inside, British outside. Both are now acceptable. I"m fine with either, but agree we should choose one and put it in style guide.